hoaxy/commands/site.py

```diff
@@ -308,6 +310,16 @@ class SiteCmd(HoaxyCommand):
         hoaxy site --enable --name=abcd.com
         (Restart of tracking processes)
 
+       Or enable list of sites:
+        hoaxy site --bulk-enable --names=abcd.com, cnn.com
+        hoaxy site --bulk-enable --exclusive --names=abcd.com, cnn.com (if exclusive is specified, all the existing sites
```

Comment:
`--exclusive` should be optional.

Reply:
fixed in 4dcc4e1133e9760ea29c45de10398b54c6f755fb